incubator-tools/cer_wer/cer_wer.ipynb

```diff
@@ -0,0 +1,296 @@
+{
```

Comment:
<p>Most of this download from GCS code should be brought out into the utilities</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/700/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>It is possible only for json files, the variable <code>groundtruth_files</code> are text files and cannot be downloaded with our functions in utilities. This is an exception  so we decided not to include the text downloading function to our utilities module.</p>